lua/entities/gmod_wire_hsranger/init.lua

```diff
@@ -18,7 +18,9 @@ local MODEL = Model( "models/jaanus/wiretool/wiretool_range.mdl" )
 	5 = SkewY
 	6 = Trace Water
 	
-	7 = unused

```

Comment:
Hmm, did it say unused for a reason?
Was there something else in slot 7 in the past, maybe?
Or is this a zero terminator?